frappe/public/js/frappe/list/list_sidebar.html

```diff
@@ -56,7 +56,7 @@
 				</a>
 				<ul class="dropdown-menu list-stats-dropdown" role="menu">
 					<div class="dropdown-search">
-						<input type="text" placeholder="Search" data-element="search" class="form-control input-xs">
+						<input type="text" placeholder={{__("Search") }} data-element="search" class="form-control input-xs">
```

Comment:
```suggestion
						<input type="text" placeholder={{ _("Search") }} data-element="search" class="form-control input-xs">
```

Is this template processed client-side or server-side? If server-side, the translate function is the one in python with a single underscore.

Reply:
here should be __() double underscore as it is js code related instead of _()?